composer.json

```diff
@@ -45,7 +45,8 @@
         "typo3/cms-install": "^9.5 || ^10.0 || 10.*@dev",
         "typo3/cms-rte-ckeditor": "^9.5 || ^10.0 || 10.*@dev",
         "typo3/cms-seo": "^9.5 || ^10.0 || 10.*@dev",
-        "scssphp/scssphp": "^1.0.6"
+        "scssphp/scssphp": "^1.0.6",
+        "typo3fluid/fluid": "^2.6"
```

Comment:
Can you please change the dependency to `^2.5.2` which is used for 9.5.0.

Reply:
I would avoid such change and keep the dependency through typo3. eg. this  change prohibits the usage of fluid 3.0

Reply:
As there is a hard dependency for this in this extension, it should be listed here as well so I would suggest to do a 
```
"typo3fluid": "^2.5.2 || ^3.0" 
``

or something depending on which fluid version works with this.

Reply:
Thx for the hint @georgringer, so your new suggestion is the way to go @tomasnorre. Omitting a dependency is imho not smart...

Reply:
Benji just told me, BP is not fully compatible with Fluid 3.x. So I'll merge it now and we'll fix it for Fluid 3.x later. Thanks you both!